module/metrics/network.go

```diff
@@ -64,6 +66,20 @@ func NewNetworkCollector() *NetworkCollector {
 			Help:      "duration [seconds; measured with float64 precision] of how long a message spent in the queue before delivered to an engine.",
 			Buckets:   []float64{0.01, 0.1, 0.5, 1, 2, 5}, // 10ms, 100ms, 500ms, 1s, 2s, 5s
 		}, []string{LabelPriority}),
+
+		outboundConnectionCount: promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
the metrics is a simple Gauge metrics since:
1. The count of connections can increment or decrement (hence not a counter but Prometheus gauge)
2. No labels are necessary since by default I am seeing these lables being added: `instance` which gives the IP address, `role` which gives the node role (I am not 100% sure where these labels get added though :( ...they seem to show up on Grafana)
e.g. network_queue_message_queue_size{**instance**="10.138.0.100:8080", job="flow_app", network="mainnet2", num="007", priority="10", **role**="collection"}